test/opacity_comparison_funcs.jl

```diff
@@ -141,14 +121,16 @@ function HI_coefficient(λ, T, Pₑ, H_I_ion_energy = 13.598)
     end
 
     ff_coef = begin
-        χs = [H_I_ion_energy, -1.0]
-        Us = [T -> 2.0, T -> 1.0] # assumption in the approach described by Gray (2005)
+        χs = [H_I_ion_energy, NaN, NaN]
```

Comment:
They aren’t, in the sense that they aren’t in the default ionization energy dict. My reasoning was that if I made a mistake and those values got used, it would be easier to catch. Do you think it’s ok for test code?

Reply:
Sounds logical to me